puzzlehq/puzzle-sdk
# jon / split out react, injected wallet

## Description
This PR:
- Adds interface to interact with new injected extension wallet (and eventual in-app browser)
- Sets minify to false when building sdk
- Splits out react from the sdk
  - we now have `@puzzlehq/sdk` and `@puzzlehq/sdk-core` 

## Ticket
n/a

## Steps to test core sdk
- Start core example
  - https://github.com/puzzlehq/puzzle-sdk-core-example
- Test extension from https://github.com/puzzlehq/firenze/pull/391
- Test chrome store extension build
- Test iOS
  - Known issue on fetching records + events @DarvishKamaliaDL to investigate 

## Steps to test react sdk
- See steps in firenze https://github.com/puzzlehq/firenze/pull/391
- Test local devtools using iOS
- Test local devtools using chromestore build

## Technicals
n/a

## Associated PRs
https://github.com/puzzlehq/firenze/pull/391
